src/_P005_DHT.ino

```diff
@@ -4,27 +4,15 @@
 //######################## Plugin 005: Temperature and Humidity sensor DHT 11/22 ########################
 //#######################################################################################################
 
+#include "src/PluginStructs/P005_data_struct.h"
 
 #define PLUGIN_005
```

Comment:
Think it's missing the `#ifdef USES_P005` here

Reply:
Yep it was indeed missing.